### PR TITLE
DDCE 5217: Play 3.0 Migration

### PR DESCRIPTION
**Pull request by EdCope**
- Migration from Play 2.8 to Play 3.0
- Upgrade to Scala 2.13.13 and updated other dependencies

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -2,21 +2,18 @@ import sbt.*
```
Comment (PaulC-TDS) [inline project/AppDependencies.scala:2]: `import sbt.*` can be replaced by `import sbt._`

File: project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=1.8.3
\ No newline at end of file
+sbt.version=1.9.7
```
Comment (PaulC-TDS) [inline project/build.properties:1]: 1.9.9 is available if you're able to upgrade this

File: project/AppDependencies.scala (hunk 1 above)
Comment (EdCope) [reply project/AppDependencies.scala:2]: Discussed about this may be helpful to leave for Scala 3

File: project/build.properties (hunk 2 above)
Comment (EdCope) [reply project/build.properties:1]: We decided to keep this at 1.9.7 to match platform recommendation on TC, so we agreed to keep on this repo too

**Review by PaulC-TDS — APPROVED** [review]

**Pull request merged**